src/test/kotlin/uk/gov/dluhc/printapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -36,10 +50,10 @@ class LocalStackContainerConfiguration {
         fun getInstance(): GenericContainer<*> {
             if (container == null) {
                 container = GenericContainer(
-                    DockerImageName.parse("localstack/localstack:1.1.0")
+                    DockerImageName.parse("localstack/localstack:1.2.0")
```

Comment:
I needed to update the version to 1.2.0, to be able to see the actual persisted test data by AWS NoSql Workbench. 

There is a known bug for the 1.1.0 version, which prevents NoSql Workbench to show table contents: 
https://github.com/localstack/localstack/issues/5737